oauth2/fosite_store_helpers.go

```diff
@@ -443,23 +443,23 @@ func testHelperFlushTokens(x InternalRegistry, lifespan time.Duration) func(t *t
 			require.NoError(t, err)
 		}
 
-		require.NoError(t, m.FlushInactiveAccessTokens(ctx, time.Now().Add(-time.Hour*24)))
+		require.NoError(t, m.FlushInactiveAccessTokens(ctx, time.Now().Add(-time.Hour*24), 100, 10))
```

Comment:
we should also ensure that batch processing is working with proper tests which take into account residual records after processing delete with request, for example

Reply:
does not seem an easy task, maybe someone will help in the future